prep-javascript-variables/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Peter Nguyen';
+    console.log('value of fullName:', 'Peter Nguyen');
+
+    var numbersOfPets = 'let x = 3';
+    console.log('vaule of numberOfPeters:', 3);
+
+    var likesVegetables = true;
+    console.log('vaule of likesVetables:', true);
+
+    var nothing = 'null';
+    console.log('vaule of nothing:', null);
+
+    var never;
+    console.log('value of never:', undefined);
```

Comment:
Rather than writing the original value again, you can pass your variable by name in all of these.

for example
```js
console.log('value of numberOfPets:', numberOfPets);
```